workflow-casava/pom.xml

```diff
@@ -29,5 +29,32 @@
             <version>2.4</version>
         </dependency>
     </dependencies>
-    
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.pyx4j</groupId>
+                <artifactId>maven-junction-plugin</artifactId>
```

Comment:
You put it back?

Reply:
I have removed this plugin and updated the workflow to use parent pom version 2.4.8 (which does not include maven-junction-plugin).